x-pack/plugins/viz_editor/public/pseudo_plugins/pie_chart_plugin/pie_chart_plugin.tsx

```diff
@@ -161,7 +160,7 @@ function getSuggestionsForField(
         q1: {
           datasourceRef,
           select: [
-            { ...firstOperation, alias: field.name },
+            { operation: 'column', alias: field.name, argument: { field: field.name } },
```

Comment:
This breaks suggestions. Please instead modify `field_config.ts` to put `column` first if you believe it is the default operation.